main/templates/components/navbar.html

```diff
@@ -0,0 +1,38 @@
+{% load static %}
+<link rel="stylesheet" type="text/css" href="{% static 'styles/components/navbar.css' %}">
```

Comment:
Well done! I always add the styles to the master layout, but this is actually a better choice!